### PR TITLE
Controller/course

### PR DESCRIPTION
**Pull request by shivamvijaywargi**
Add Makefile, fix module not found error, comment out upload file function for now

### CODE REVIEW

**Comment by sub1120** [comment]
@shivamvijaywargi, it worked. Not sure how, I think following steps helped,
 - Reopned my terminal in vscode
 - Down the container
 - Up the container 

I have test S3 and cloudinary both, its working fine. Please review the code.

**Comment by sub1120** [comment]
Once it is merged, I will announce in discussion about this, so other can use the same.

File: src/configs/index.ts [hunk 1]
```diff
@@ -26,6 +26,10 @@ const CONFIG = {
     EMAIL_FROM: process.env.SMTP_EMAIL_FROM || "",
   },
   VERIFY_URL: process.env.VERIFY_URL || "http://localhost:3000/verify-email",
+  CLOUDINARY_CLOUD_NAME: process.env.CLOUDINARY_CLOUD_NAME,
+  CLOUDINARY_API_KEY: process.env.CLOUDINARY_API_KEY,
+  CLOUDINARY_SECRET_KEY: process.env.CLOUDINARY_SECRET_KEY,
+  S3_BUCKET_REGION: process.env.S3_BUCKET_REGION,
```
Comment (shivamvijaywargi) [inline src/configs/index.ts:32]: We do not have other secrets in here, are they not required?

**Review by shivamvijaywargi — COMMENTED** [review]
Please check the comment

File: src/configs/index.ts (hunk 1 above)
Comment (sub1120) [reply src/configs/index.ts:32]: Yes, AWS credentials are automatically configured by AWS SDK, we just need to add in .env.

Comment (shivamvijaywargi) [reply src/configs/index.ts:32]: Cool

Comment (shivamvijaywargi) [reply src/configs/index.ts:32]: I am unable to merge since you might need to resolve your comment I believe

Comment (sub1120) [reply src/configs/index.ts:32]: Okay, I will merge now.

**Review by sub1120 — APPROVED** [review]

**Pull request merged**